streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -903,20 +897,15 @@ public void shouldTriggerRecordingOfRocksDBMetricsIfRecordingLevelIsDebug() {
             streams.start();
         }
 
-        PowerMock.verify(Executors.class);
```

Comment:
please add `verify` to make sure `rocksDBMetricsRecordingTriggerThread` is used to create thread in this test.